modules/prebidServerBidAdapter/config.js

```diff
@@ -12,6 +12,9 @@ export const S2S_VENDORS = {
     enabled: true,
     endpoint: '//prebid-server.rubiconproject.com/openrtb2/auction',
     syncEndpoint: '//prebid-server.rubiconproject.com/cookie_sync',
-    timeout: 500
+    timeout: 500,
+    adapterOptions: {
+      rubicon: { singleRequest: false }
```

Comment:
shouldn't the Rubicon PBS adapter handle this default setting rather than having a bidder-specific setting hard coded into the config for everyone, even when the bidder is not being used?

Reply:
I agree and Brett confirmed that the Rubicon PBS server will default to this value. Fix committed.